### PR TITLE
Update Debian Packaging

### PR DESCRIPTION
**Pull request by aazam-gh**
Updated files in the `zthindeb` and `sdkdeb` directories
Updated `buildzthindeb` and `buildzvmsdkdeb` script files

### CODE REVIEW

**Comment by mfriesenegger** [comment]
I am able to successfully build zthin and zvmsdk manually on my ubuntu l1cc instance upgraded to 24.04.  Two fixes need to be applied to the [aazam-gh:deb](https://github.com/aazam-gh/build-zvmsdk/tree/deb) branch.

There is a typo in https://github.com/aazam-gh/build-zvmsdk/blob/87b3b583a6a7e1fc45e53f2b591b27e242f6f618/sdkdeb/control#L14.  `python zthin` causes a fatal build error.  The line should be
`Depends: python3 (>= 3.12), python3-netaddr (>= 0.7.5), python3-jwt(>= 1.0.1), python3-mock, python3-requests(>= 2.6.0), python3-routes(>= 2.2), python3-webob(>= 1.2.3), python3-jsonschema(>= 2.3.0), python3-six(>= 1.9.0), python3-jinja2, zthin (>= 3.1.0), apache2, libapache2-mod-wsgi-py3`

I also suggest changing https://github.com/aazam-gh/build-zvmsdk/blob/87b3b583a6a7e1fc45e53f2b591b27e242f6f618/buildzvmsdkdeb#L54 to stop the tests during the build process.  The new line should be
`DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -uc -us`

Here are the repeatable steps after making these two changes to the deb branch so that packages can be manually built on ubuntu on l1cc

1. As the linux1 user, use sudo to update ubuntu 22.04 LTS on l1cc to 24.04
2. sudo apt install build-essential debhelper dh-python dh-make python3-all python3-webob python3-mock python3-netaddr python3-routes python3-testtools libtirpc-dev
3. As linux1 user, git clone https://github.com/aazam-gh/build-zvmsdk.git
4. cd ~/build-zvmsdk/
5. git checkout -b deb origin/deb
6. bash ./buildzthindeb master
7. cp feilong/zthin_3.1.2_s390x.deb ~  (otherwise the package will be deleted when buildzvmsdkdeb is run)
8. bash ./buildzvmsdkdeb master
9. cp zvmsdk_1.4.0_all.deb ~  (I think the zvmsdk_1.4.0_all.deb file is in ~/build-zvmsdk/ but it might be in ~/build-zvmsdk/feilong)

Please confirm that this also works in your l1cc ubuntu.

**Comment by aazam-gh** [comment]
Confirmed changes work as intended

**Comment by aazam-gh** [comment]
PR ready to be merged

File: vhost_deb.config [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <Directory /usr/lib/python3.12/site-packages/zvmsdk/sdkwsgi>
+        Require all granted
+    </Directory>
+
+    <Directory /usr/bin>
+    <Files zvmsdk-wsgi>
+        Require all granted
+    </Files>
+    </Directory>
+</VirtualHost>
```
Comment (Bischoff) [inline vhost_deb.config]: Add missing final linefeed please

**Review by Bischoff — CHANGES_REQUESTED** [review]
+ just one nitpick about a missing linefeed
+ please also add the `a2enmod zvmsdk` after you instal the vhost file

thanks!

File: vhost_deb.config (hunk 1 above)
Comment (aazam-gh) [reply vhost_deb.config]: Done

**Review by Bischoff — APPROVED** [review]
lgtm now

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**